apps/system/utils/utils_ps.c

```diff
@@ -133,12 +133,17 @@ static void ps_print_values(char *buf, void *arg)
 	if (cpu_idx != cpu && cpu != CONFIG_SMP_NCPUS) {
 		return;
 	}
-#endif
-	
+
 	printf("%5s | %4s | %4s | %7s | %c%c | %8s | %3s", stat_info[PROC_STAT_PID], stat_info[PROC_STAT_PRIORITY], \
 		flags & TCB_FLAG_ROUND_ROBIN ? "RR  " : "FIFO", utils_ttypenames[(flags & TCB_FLAG_TTYPE_MASK) >> TCB_FLAG_TTYPE_SHIFT], \
 		flags & TCB_FLAG_NONCANCELABLE ? 'N' : ' ', flags & TCB_FLAG_CANCEL_PENDING ? 'P' : ' ', \
 		utils_statenames[state], stat_info[PROC_STAT_CPU]);
+#else
+	printf("%5s | %4s | %4s | %7s | %c%c | %8s ", stat_info[PROC_STAT_PID], stat_info[PROC_STAT_PRIORITY], \
+		flags & TCB_FLAG_ROUND_ROBIN ? "RR  " : "FIFO", utils_ttypenames[(flags & TCB_FLAG_TTYPE_MASK) >> TCB_FLAG_TTYPE_SHIFT], \
+		flags & TCB_FLAG_NONCANCELABLE ? 'N' : ' ', flags & TCB_FLAG_CANCEL_PENDING ? 'P' : ' ', \
+		utils_statenames[state]);
+#endif
```

Comment:
Let's reduce duplicated codes.
```
	printf("%5s | %4s | %4s | %7s | %c%c | %8s", stat_info[PROC_STAT_PID], stat_info[PROC_STAT_PRIORITY], \
		flags & TCB_FLAG_ROUND_ROBIN ? "RR  " : "FIFO", utils_ttypenames[(flags & TCB_FLAG_TTYPE_MASK) >> TCB_FLAG_TTYPE_SHIFT], \
		flags & TCB_FLAG_NONCANCELABLE ? 'N' : ' ', flags & TCB_FLAG_CANCEL_PENDING ? 'P' : ' ', \
		utils_statenames[state]);

#ifdef CONFIG_SMP
	cpu_idx = atoi(stat_info[PROC_STAT_CPU]);
	if (cpu_idx != cpu && cpu != CONFIG_SMP_NCPUS) {
		return;
	}

	printf(" | %3s", stat_info[PROC_STAT_CPU]);
#endif
```

1. You should remove a space after `%8s`.
2. You can keep #else code only, please find above.

And this is a change in ps, not in cpuload. Let's split a commit.

Reply:
Instead of splitting, let's modify the title and description.